Doc/library/pydoc.rst

```diff
@@ -65,18 +65,18 @@ manner similar to the Unix :program:`man` command.  The synopsis line of a
 module is the first line of its documentation string.
 
 You can also use :program:`pydoc` to start an HTTP server on the local machine
-that will serve documentation to visiting web browsers.  :program:`pydoc -p 1234`
+that will serve documentation to visiting web browsers. :program:`python -m pydoc -p 1234`
```

Comment:
Please leave the leading space alone. reStructuredText allows for one or two spaces after a period.

Reply:
Yes will do